tidy3d/components/data/data_array.py

```diff
@@ -220,15 +219,7 @@ def values(self):
         np.ndarray
             The values of the DataArray.
         """
-        if isbox(self.data):
-            warnings.warn(
-                "'DataArray.values' for automatic differentiation is deprecated, "
-                "please use 'DataArray.data' instead.",
-                DeprecationWarning,
-                stacklevel=2,
-            )
-            return self.data
-        return super().values
+        return self.data if isbox(self.data) else super().values
```

Comment:
so, no side effects internally from doing this? I think we discussed you were worried about it?

Reply:
yeah this is fine. `values` simply does a `np.asarray(self.data)`, which we don't need to do here because if `data` is an `ArrayBox`, then it already is (or was) a numpy array and no casting is necessary.

Reply:
sounds good